### PR TITLE
create html file

### PR DESCRIPTION
**Pull request by KirylKarzhavin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<h1>Hello world!</h1>
```
Comment (newqwes) [inline index.html:10]: It's can't be so simple

**Comment by newqwes** [comment]
some text

File: style.css [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    color: #6e6e6e;
+    margin: 0px;
+}
+
+h4 
+{
+    color: #3eb5f1;
+}
+
+p
```
Comment (newqwes) [inline style.css:52]: Is it really need there?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+textarea 
+{
+    text-align: left;
+    resize: none;
+    width: 100%;
+    height: 100px;
+}
+
+@import url("https://fonts.googleapis.com/css?family=Barlow|Tomorrow:400,700&display=swap");
```
Comment (newqwes) [inline style.css:162]: Try to import fonts in html file

File: index.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link href="style.css" rel="stylesheet" type="text/css">
```
Comment (newqwes) [inline index.html:4]: Please change settings from `1 tab` to `2 space` for indent

File: style.css [hunk 5]
```diff
@@ -0,0 +1,164 @@
+body {
+    background-color: white;
+    margin: 2%;
+    font-family: Tomorrow, sans-serif;
+} 
+
+.parrent-container
```
Comment (newqwes) [inline style.css:7]: Do not wrap the curly brace on a new line. Follow the same style

File: index.html [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (newqwes) [inline index.html:6]: Forgot space

File: index.html (hunk 4 above)
Comment (KirylKarzhavin) [reply index.html:4]: fixed.

File: style.css (hunk 5 above)
Comment (KirylKarzhavin) [reply style.css]: fixed.

File: index.html (hunk 6 above)
Comment (KirylKarzhavin) [reply index.html]: fixed.

File: index.html (hunk 4 above)
Comment (newqwes) [reply index.html:4]: still seems like with big indent...

**Pull request merged**